examples/apitest.py

```diff
@@ -41,21 +40,14 @@ def main():
     )
     print(chat_completion)
 
-    print("TESTING COMPLETION API")
```

Comment:
no completion models work at openai anymore, so moved it from this integration test.